package.json

```diff
@@ -19,15 +19,6 @@
     "prepare": "npm run build"
   },
   "dependencies": {
-    "@types/history": "^4.6.2",
-    "@types/lodash": "^4.14.88",
-    "@types/node": "^8.0.54",
-    "@types/react": "^16.0.27",
-    "@types/react-dom": "^16.0.3",
-    "@types/react-hot-loader": "^3.0.5",
-    "@types/react-redux": "^5.0.14",
-    "@types/redux": "^3.6.0",
-    "@types/redux-devtools-extension": "^2.13.2",
     "history": "^4.7.2",
```

Comment:
Better to keep these here:
https://stackoverflow.com/questions/45176661/how-do-i-decide-whether-types-goes-into-dependencies-or-devdependencies

Reply:
Ok i see i will rollback this change then

Reply:
fixed